from0k2bp.tex

```diff
@@ -552,11 +562,11 @@
 the aforementioned single bit.
 \end{quote}
 
-For more details on this basic (if rather bizarre at first) reasoning
-for the construction of zero knowledge proofs, refer to e.g. ,
+For more details on this basic reasoning (rather bizarre at first) 
+for the construction of zero knowledge proofs, refer to e.g. 
 {[}\protect\hyperlink{anchor-26}{14}{]} for a discursive introduction,
 and {[}\protect\hyperlink{anchor-27}{17}{]} for an in-depth discussion
-with academic rigour (hat tip to Jonas Nick for pointing me at that!).
+with academic rigour\footnote{I have to tip Jonas Nick for pointing me at that!}.
```

Comment:
"hat tip" is fine here (re: footnotes see earlier comment), if this was an academic document I'd have to go with a more boring "thanks to" or "acknowledgement", but it isn't, and this is a well known colloquial expression by now.

Reply:
I see. Fixed.